src/useq/_mda_sequence.py

```diff
@@ -388,6 +394,7 @@ def iter_sequence(
     base_event_kwargs: MDAEventDict | None = None,
     event_kwarg_overrides: MDAEventDict | None = None,
     position_offsets: PositionDict | None = None,
+    shutter_plan: ShutterOpenAxes | None = None,
```

Comment:
already add stuff to our nice and trimmed down signature? ;)

this isn't necessary.  see note below

Reply:
Actually I found an autofocus bug in the code and I think we should add a `autofocus_plan_override` argument to the `iter_sequence` method. I opened this PR  #98  to explain it in detail.

Reply:
I still don't think so but I'll have a look.  We have kwargs for both fallbacks and overrides ... so it's hard to see how we
Need more. (It's possible you need to change the arguments passed in to either the fallbacks or overrides, but I still don't think we need more kwargs)

Reply:
Yeah I think what you want to do is use sequence.replace on the sequence being passed.  actually, I think we could do that for all the overrides and possibly the fallbacks too come to think of it.  Getting us back to just a single argument